### DISCUSSION TITLE
Login for MiniO Console?

### DISCUSSION THREAD

**Discussion by MCKLtech** [beyondcode/herd-community#507] [Q&A]
I've just started a MiniO instance and for the console @ http://127.0.0.1:9000 the default login (minioadmin) does not work.

Is there a specific login for Herd?

**Comment by rginnow** [comment]
For anyone else looking, this is documented here: https://herd.laravel.com/docs/1/herd-pro-services/minio

> Log in with the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to create your first bucket and use the object storage like in your production environment.
